doc/tools/coqrst/coqdomain.py

```diff
@@ -42,6 +46,20 @@
 from .notations.sphinx import sphinxify
 from .notations.plain import stringify_with_ellipses
 
+# FIXME: Patch this in Sphinx
+# https://github.com/coq/coq/issues/12361
+def visit_desc_signature(self, node):
+    hyper = ''
+    if node.parent['objtype'] != 'describe' and node['ids']:
+        for id in node['ids']:
+            hyper += self.hypertarget(id)
+    self.body.append(hyper)
+    if not node.get('is_multiline'):
+        self._visit_signature_line(node)
+    else:
+        self.body.append('%\n\\pysigstartmultiline\n')
+LaTeXTranslator.visit_desc_signature = visit_desc_signature
+
```

Comment:
I don't understand this part.

Reply:
Does the linked issue help?

Reply:
(It's a minor modification of the code that's in Sphinx already: I'm iterating over all ids instead of just the first one)